CineElorrietaMVC/src/main/java/Controlador/ControladorPanelRegistro.java

```diff
@@ -0,0 +1,120 @@
+package Controlador;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import Modelo.Modelo;
+import Vista.PanelRegistro;
+import Vista.Vista;
+
+public class ControladorPanelRegistro {
+
+	@SuppressWarnings("unused")
+	private Modelo modelo;
+	private Vista vista;
+	@SuppressWarnings("unused")
+	private Controlador controlador;
+	private PanelRegistro panelRegistro;
+
+	public ControladorPanelRegistro(Modelo modelo, Vista vista, Controlador controlador) {
+		this.modelo = modelo;
+		this.vista = vista;
+		this.controlador = controlador;
+	}
+
+	public void mostrarPanelRegistro() {
+		this.panelRegistro = new PanelRegistro(this);
+		this.vista.mostrarPanel(this.panelRegistro);
+
+	}
+
+	public void accionadoBottonVolverPanelLogin() {
+		this.controlador.navegarPanelLogin();
+	}
+
+	public boolean comprobarBBDDnif(String nif) {
+		return this.modelo.getConexion().comprobarSiExisteNIF(nif);
+	}
+
+	public boolean comprobarBBDDdni(String dni) {
+		return this.modelo.getConexion().comprobarSiExisteDNI(dni);
+	}
+
+	public void insertarRegistro(String DNI, String Nombre, String Apellido, String contrasena, String nif) {
+		this.modelo.getConexion().insertarRegistro(DNI, Nombre, Apellido, contrasena, nif);
+
+	}
+
+	public boolean comprobarNif(String nif) {
```

Comment:
Este método también lo usáis en facturas, unificarlo en una clase.

Reply:
Corregido.

Reply:
Corregido.